common/djangoapps/third_party_auth/strategy.py

```diff
@@ -34,3 +34,29 @@ def setting(self, name, default=None, backend=None):
         # At this point, we know 'name' is not set in a [OAuth2|LTI|SAML]ProviderConfig row.
         # It's probably a global Django setting like 'FIELDS_STORED_IN_SESSION':
         return super(ConfigurationModelStrategy, self).setting(name, default, backend)
+
+    def request_host(self):
+        """
+        Host in use for this request
+        """
+        # TODO: this override is a temporary measure until upstream python-social-auth patch is merged:
```

Comment:
We may consider holding off mergining this upstream until the PSA patch is applied – I wouldn't expect this to take ages.